HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -419,12 +419,19 @@ def self.create_return_air_duct_zone(model, loop_name)
     return ra_duct_zone
   end
 
-  def self.create_sens_lat_load_actuator_and_equipment(model, name, space, frac_lat, frac_lost, is_duct_load_for_report = nil)
+  def self.create_sens_lat_load_actuator_and_equipment(model:, name:, space:, frac_lat:, frac_lost:, hpxml_fuel_type: nil, end_use: nil, is_duct_load_for_report: nil)
```

Comment:
change to the other equipment object creation to include optional fuel and end use inputs, probably better to rename it as create_other_equipment_object_and_actuator?